pkg/controller/disruption/disruption.go

```diff
@@ -381,14 +381,14 @@ func (dc *DisruptionController) Run(stopCh <-chan struct{}) {
 
 func (dc *DisruptionController) addDb(obj interface{}) {
 	pdb := obj.(*policy.PodDisruptionBudget)
-	klog.V(4).Infof("add DB %q", pdb.Name)
+	klog.V(4).InfoS("Add DB", "pdb", pdb.Name)
```

Comment:
api objects are supposed to be logged with `klog.KObj()`, right?

```suggestion
	klog.V(4).InfoS("Add DB", "pdb", klog.KObj(pdb))
```

(likewise throughout)

Reply:
Correct, fixed!